Throughput Documentation Update (#6611)
Particular/docs.particular.net: nservicebus/throughput-tool/index.md

---
title: Endpoint throughput tool
summary: Use the Particular endpoint throughput counter tool to measure the usage of an NServiceBus system.
reviewed: 2024-05-22
related:
  - nservicebus/usage
---

> [!NOTE]
> The recommended method of measuring the throughput of an NServiceBus system is via [ServicePulse](./../../servicepulse/usage.md).

> > The endpoint throughput counter tool is offered as an alternate option for customers constrained in running ServiceControl.

Reviewer: ```suggestion
> The endpoint throughput counter tool is offered as an alternate option for customers that aren't able to use ServicePulse.
```
